### PR TITLE
Moving Example Configs to a Dedicated Folder, adding STILTS example

### PR DESCRIPTION
**Pull request by pruksmhc**
Changes:
* Adding examples for STILTS
* Moving examples configs to example folder

### CODE REVIEW

File: config/examples/stilts_example/mrpc_mnli_phase1.conf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+include "../../defaults.conf"
+
+// For STILTS training, first run this experiment, before 
+// mrpc_mnli_phase2.conf.In this STILTS experiment, we add first train on MRPC,
```
Comment (sleepinyourhat) [inline config/examples/stilts_example/mrpc_mnli_phase1.conf]: Fix spacing

Comment (sleepinyourhat) [reply config/examples/stilts_example/mrpc_mnli_phase1.conf]: Why MRPC then QQP then MRPC? This doesn't make sense (MRPC is way too small to be useful for pretraining), and it doesn't match the filename...

File: config/examples/stilts_example/mrpc_mnli_phase2.conf [hunk 2]
```diff
@@ -0,0 +1,17 @@
+include "./bert_mrpc.conf"
+
+// First, run mrpc_mnli_phase1.conf first, then this config file.
```
Comment (sleepinyourhat) [inline config/examples/stilts_example/mrpc_mnli_phase2.conf]: first ... first

File: config/examples/stilts_example/mrpc_mnli_phase1.conf (hunk 1 above)
Comment (sleepinyourhat) [reply config/examples/stilts_example/mrpc_mnli_phase1.conf]: Maybe replace this with something like MNLI => RTE? That was the simplest success case for STILTs.

If you'd rather stick closer to this, maybe switch to just BERT+QQP => MRPC, which is at least fairly likely to help on the target task.

Comment (pruksmhc) [reply config/examples/stilts_example/mrpc_mnli_phase1.conf]: Just to make sure, this is RTE pretraining, then intermediate task of MNLI, and then finetuning on RTE?

Comment (sleepinyourhat) [reply config/examples/stilts_example/mrpc_mnli_phase1.conf]: Simper: Train BERT on MNLI, then on RTE.

Comment (sleepinyourhat) [reply config/examples/stilts_example/mrpc_mnli_phase1.conf]: Probably doable with a single conf.

Comment (W4ngatang) [reply config/examples/stilts_example/mrpc_mnli_phase1.conf]: Definitely should be a single run / configuration. Something like:
`pretrain_tasks = mnli`
`target_tasks = rte`
Would do it.

File: config/examples/copa_bert.conf [hunk 3]
```diff
@@ -0,0 +1,37 @@
+// An example configuration for the CommitmentBank task with BERT.
```
Comment (sleepinyourhat) [inline config/examples/copa_bert.conf]: CommitmentBank != COPA

File: config/examples/stilts_example.conf [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+//BERT-specific parameters
+bert_embeddings_mode = "top"    
+bert_fine_tune = 1
+sep_embs_for_skip = 1
+sent_enc = "none"
+classifier = log_reg // following BERT paper
+
+dropout = 0.1 // following BERT paper
+optimizer = bert_adam
+max_epochs = 1
```
Comment (sleepinyourhat) [inline config/examples/stilts_example.conf]: Should be 3 or 4 for standard BERT training.

File: config/examples/stilts_example.conf [hunk 5]
```diff
@@ -0,0 +1,38 @@
+// This config is for STILTS training for BERT -> MNLI -> RTE
```
Comment (sleepinyourhat) [inline config/examples/stilts_example.conf]: Mention that we're using base. Most users will want to mess with this then switch to large.

**Pull request merged**